hypoddpy/hypodd_relocator.py

```diff
@@ -384,9 +384,9 @@ def _write_catalog_input_file(self, phase_weighting=None):
                         station_id=pick["station_id"])
                     self.log(msg, level="warning")
                     continue
-                weight = phase_weighting(pick['station_id'], pick['phase'],
+                weight = phase_weighting((pick['station_id'], pick['phase'],
                                          pick['pick_time'],
-                                         pick['pick_time_error'])
+                                         pick['pick_time_error']))
```

Comment:
Yes, it does seem true that the lambda function you defined is taking in a tuple, I got an error:

```python
phase_weighting = lambda (sta_id, ph_type, time, uncertainty): 1.0
phase_weighting(1, 2, 3, 4)
```

This is the reason I update it to input a tuple.